launch_testing_ros/launch_testing_ros/test_runner.py

```diff
@@ -19,19 +19,4 @@
 
 
 class LaunchTestRunner(launch_testing.test_runner.LaunchTestRunner):
```

Comment:
We should consider deprecating launch_testing_ros

Reply:
I thought about it, but I chose not to do that now both to keep this PR small and to not take that leap and remove a package we may need again afterwards.